### DISCUSSION TITLE
Use PSAT cli tool to scan mutiple pages, is there anyway to scan the sitemap and show the combination resault.

### DISCUSSION THREAD

**Discussion by Joeyindigital** [GoogleChromeLabs/ps-analysis-tool#707] [Support Forum] [answered]
Hi I am running an websit cookie audit for a client. While running the psat cli tool to scan multiple pages , is there a way to show the comnination resault instead of mutiple url files?

**Comment by milindmore22** [comment, marked as answer]
Hello @Joeyindigital,

Thank you for your valuable suggestion! We appreciate you reaching out.

We understand that you are looking for an aggregated report for sitemaps. This feature is already on our development roadmap, and you can track its progress on GitHub issue #627.

We’re excited to bring this functionality to PSAT soon!
